sql-cli/include/base/.airflow/.dags/.sql/example_basic_transform/top_animations.sql

```diff
@@ -0,0 +1,5 @@
+SELECT Title, Rating
```

Comment:
@feluelle shouldn't these SQL example workflows be within `include/base/workflows` for now, until we make examples optional (0.2)?

I'd expect only the materialised SQL files to be in hidden .airflow folder

Reply:
These are the materialised ones.